v2/components/android-breakage-modal-wrapper.jsx

```diff
@@ -0,0 +1,136 @@
+
+        const deltaX = Math.abs(event.clientX - obj.current.startX)
+        const deltaY = Math.abs(event.clientY - obj.current.startY)
+
+        // If the movement exceeds the threshold, it is not a tap
+        if (deltaX > tapThreshold || deltaY > tapThreshold) {
+            obj.current.isTap = false
+        }
+    }
+
+    function onPointerUp() {
+        if (!obj.current.isTap) return
+        /** @type {import("../custom-elements/android-breakage-dialog").AndroidBreakageDialog | null} */
+        const elem = document.querySelector(DDG_DIALOG_NAME)
+        if (!elem) return console.warn('could not find custom element', 'ddg-android-breakage-dialog')
+        if (!selectRef.current) return console.warn('could not find select ref')
+        elem.show(selectRef.current.value)
+    }
+
+    return (
+        <>
+            <CustomElementLoader src={DDG_DIALOG_PATH} element={DDG_DIALOG_NAME}>
+                <AndroidBreakageDialogWrapper
+                    items={randomised}
+                    onSelect={(value) => {
+                        if (!selectRef.current) return
+                        selectRef.current.value = value
+                    }}
+                />
+            </CustomElementLoader>
+            <div
+                className="form__select breakage-form__input--dropdown"
+                onPointerDown={onPointerDown}
+                onPointerMove={onPointerMove}
+                onPointerUp={onPointerUp}
+                data-testid="select-click-capture"
+            >
+                <select name="category" ref={selectRef} style={{ pointerEvents: 'none' }}>
+                    <option value="" selected disabled>
+                        {ns.report('pickYourIssueFromTheList.title')}
+                    </option>
+                    {randomised.map(([key, value]) => {
+                        return <option value={key}>{value}</option>
+                    })}
+                </select>
+            </div>
+        </>
+    )
+}
+
+/**
+ * @params {Parameters<AndroidModalWrapper>} props
+ */
+export function AndroidBreakageDialogWrapper({ items, onSelect }) {
+    /** @type {import("preact").RefObject<import("../custom-elements/android-breakage-dialog.js").AndroidBreakageDialog | null>} */
+    const ref = useRef(null)
+
+    useEffect(() => {
+        const controller = new AbortController()
+        ref.current?.addEventListener(
+            'did-select',
+            (/** @type {any} */ e) => {
+                const selection = /** @type {{value: string}} */ e.detail
+                const matched = items.find(([name]) => name === selection.value)
+                if (!matched) throw new Error('value did not match a variant')
+                const [value] = matched
+                onSelect(value)
+            },
+            { signal: controller.signal }
+        )
+        return () => {
+            return controller.abort()
+        }
+    }, [])
```

Comment:
the custom element emits an event when an item was selected + 'ok' was clicked. We listen here and reflect it back to the page